### PR TITLE
ci: more strictness on codeowners linter

### PR DESCRIPTION
**Pull request by busla**
# CODEOWNER cleanup

Explicitly setting codeowners to projects and have codeowners linter job fail if a file is missing so we can stop bombarding team-core with review requests when there are actual owners.

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205364219567884

### CODE REVIEW

**Review by AndesKrrrrrrrrrrr — COMMENTED** [review]
Nice :heart:

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -5,25 +5,39 @@
...
 # Organisation name is @island-is, not @island.is
 
-# Fallback. If there is no owner, assign to core maintainers.
-*                                                                                                        @island-is/core
 
 # Top level project ownership. When a project contains, for example, templates or modules
 # that are developed by others teams, we first list down the team working on the core and
 # further below the other team working on other part of the project.
+# All files in project root but not dirs
+/                                                                                                        @island-is/core
```
[the remark this answers is not in the document]
Comment (busla) [reply .github/CODEOWNERS:13]: this is still a draft

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -5,25 +5,39 @@
...
 /libs/application/                                                                                       @island-is/norda-applications
 /libs/contentful-extensions/                                                                             @island-is/juni @island-is/stefna
 /libs/service-portal/                                                                                    @island-is/hugsmidjan
 /libs/portals/admin/                                                                                     @island-is/aranja
 /libs/portals/core/                                                                                      @island-is/aranja @island-is/hugsmidjan
 
-/*                                                                                                       @island-is/core
+/.yarn/                                                                                                  @island-is/core
+/.githooks/                                                                                              @island-is/core
+/.vscode/                                                                                                @island-is/core
```
[the remark this answers is not in the document]
Comment (busla) [reply .github/CODEOWNERS:25]: this would match all directories in the root, no?

**Comment by andes-it** [comment]
This PR is stale because it has been open 15 days with no activity. Remove stale label or comment or this will be closed in 5 days.

**Comment by andes-it** [comment]
This PR's had a 15-day siesta. Lost its spark, did it? It's making an exit in 5 days unless the stale label disappears.

**Comment by andes-it** [comment]
This PR closed its doors. Sat around for a day after the stale tag, then reckoned it's not worth the fuss.

**Pull request closed without merging**